docs/building.md

```diff
@@ -0,0 +1,15 @@
+# Build from source TL;DR
+
+0. Install Go
+
+1. Get the source 
+
+        $ go get -u -d github.com/GoogleCloudPlatform/skaffold
+        package github.com/GoogleCloudPlatform/skaffold: no Go files in /path/to/go/workspace/src/github.com/GoogleCloudPlatform/skaffold
```

Comment:
Now that you added `-d`, you won't get the `no Go files` error. You can remove this line.

Reply:
Oh, fixed. Thanks!